packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```diff
@@ -33,227 +45,359 @@
-      @include bpk-hero-4;
-    }
-
-    &__subheading {
       @include bpk-subheading;
     }
-  }
 
-  @include breakpoint-portrait-medium {
-    &__sponsor-label {
+    @include bpk-breakpoint-mobile {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-heading-3;
+      @include bpk-subheading;
     }
 
-    &__tagline {
+    @include bpk-breakpoint-small-mobile {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-subheading;
+      @include bpk-label-1;
     }
+  }
 
-    &__headline {
-      margin-bottom: bpk-spacing-md();
-
+  &__headline {
+    @include bpk-breakpoint-above-tablet {
       @include bpk-hero-5;
+
+      &:not(:last-child) {
+        margin-bottom: bpk-spacing-md();
+      }
     }
 
-    &__subheading {
-      @include bpk-body-longform;
+    @include bpk-breakpoint-tablet {
+      @include bpk-heading-1;
+
+      &:not(:last-child) {
+        margin-bottom: bpk-spacing-md();
+      }
     }
-  }
 
-  @include breakpoint-portrait-small {
-    &__sponsor-label {
-      margin-bottom: bpk-spacing-sm();
+    @include bpk-breakpoint-small-tablet {
+      margin-bottom: bpk-spacing-md();
 
-      @include bpk-label-1;
+      @include bpk-heading-1;
     }
 
-    &__tagline {
+    @include bpk-breakpoint-mobile {
       margin-bottom: bpk-spacing-md();
 
-      @include bpk-label-1;
+      @include bpk-hero-5;
     }
 
-    &__headline {
+    @include bpk-breakpoint-small-mobile {
       margin-bottom: bpk-spacing-md();
 
       @include bpk-heading-2;
     }
+  }
+
+  &__subheading {
+    @include bpk-breakpoint-above-tablet {
+      @include bpk-body-default;
+    }
+
+    @include bpk-breakpoint-tablet {
+      @include bpk-body-default;
+    }
 
-    &__subheading {
+    @include bpk-breakpoint-small-tablet {
+      @include bpk-subheading;
+    }
+
+    @include bpk-breakpoint-mobile {
+      @include bpk-body-longform;
+    }
+
+    @include bpk-breakpoint-small-mobile {
       @include bpk-body-default;
     }
   }
 
-  &__container {
+  &__promo-content,
+  &__sponsor-content {
     display: flex;
     flex-direction: column;
 
-    @include breakpoint-landscape {
-      justify-content: center;
+    @include bpk-breakpoint-above-tablet {
+      width: 22.5rem;
 
       &--start {
         align-items: start;
+        text-align: start;
       }
 
       &--center {
         align-items: center;
+        text-align: center;
       }
 
       &--end {
-        align-items: end;
+        align-items: start;
+        text-align: start;
       }
     }
 
-    @include breakpoint-portrait {
-      justify-content: space-between;
+    @include bpk-breakpoint-tablet {
+      width: 20rem;
 
-      &--inverted {
-        flex-direction: column-reverse;
+      &--start {
+        align-items: start;
+        text-align: start;
+      }
+
+      &--center {
+        align-items: center;
+        text-align: center;
+      }
+
+      &--end {
+        align-items: start;
+        text-align: start;
       }
```

Comment:
Similar here, you can simplify this by only needing to define `--start`, `--center`, `--end` once and just use the breakpoints to override the width

Reply:
👍🏼